src/DataManagement/Saving/AlternativeSavingCalculator.cs

```diff
@@ -3,24 +3,23 @@
 using ePiggyWeb.DataManagement.Entries;
 using ePiggyWeb.DataManagement.Goals;
 using ePiggyWeb.Utilities;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Configuration;
 
 namespace ePiggyWeb.DataManagement.Saving
 {
-    public class AlternativeSavingCalculator
+    public class AlternativeSavingCalculator : PageModel
```

Comment:
This class doesn't need to inherit from PageModel ( PageModel is only used in the front end files)

Reply:
ah, okay, that makes more sense